src/otx/api/usecases/exportable_code/demo/requirements.txt

```diff
@@ -1,4 +1,4 @@
 openvino==2023.0
-openvino-model-api==0.1.2
-otx @ git+https://github.com/openvinotoolkit/training_extensions/@77b635f4fba0a8acca221ec7e8b1fadd734358da#egg=otx
+openvino-model-api==0.1.3
+otx==1.4.0rc4
```

Comment:
This should be commit-based on develop.

Reply:
it's not easy to understand what I need to do for this PR. should it be the specific commit based on the develop branch? does it mean that should not be the version string?

Reply:
It was agreed so far that we maintain commit-based otx install on this line on develop branch.
Usually we set the latest commit of the PR. If you want someone who update `otx.api` to update this line later, please go ahead. It was just agreement, not strict rule.